omniduct/databases/druid.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import
+
+from omniduct.utils.debug import logger
+
+from .base import DatabaseClient
+
+
+class DruidClient(DatabaseClient):
+
+    PROTOCOLS = ['druid']
+    DEFAULT_PORT = 80
+
+    def _init(self):
+        self.__druid = None
+
+    # Connection
+    def _connect(self):
+        from pydruid.db import connect
+        logger.info('Connecting to Druid database ...')
+        self.__druid = connect(self.host, self.port, path='/druid/v2/sql/', scheme='http')
+
+    def _is_connected(self):
+        return hasattr(self, '__druid') and self.__druid is not None
```

Comment:
`hasattr` doesn't seem to be necessary here, given that it is explicitly set in the __init__ method.

Reply:
udpated!